setup.cfg

```diff
@@ -83,6 +83,7 @@ install_requires =
     ansible
     pyyaml
     six
+    ruamel.yaml
```

Comment:
What is the reliance on ruamel for?

Reply:
These 2 steps were added between existing ansible-lint functionality
1. (existing) `RulesCollection.run` reads yml file into text
2. (new) `ruamel.yaml` parses text while retaining access to comments through it’s python object
3. (new) `ansiblelint.utils.add_skipped_rules` looks at those comments for any that match a rule skip, if found it adds a `skipped_rules` key to the task with a list of rule ids. `ruamel.yaml` then dumps it’s serialized yaml back into text
4. (existing) Pass text into each rule’s match method such as `matchtasks` which parses using the `yaml` and `ansible` packages (this pr doesn’t try to change any of that parsing)

We are using `yaml = ruamel.yaml.YAML()` and their [docs](https://yaml.readthedocs.io/en/latest/dumpcls.html#dumping-python-classes) state:
> Only yaml = YAML(typ='unsafe') loads and dumps Python objects out-of-the-box. And since it loads any Python object, this can be unsafe.